rpm/podman.spec

```
@@ -242,6 +248,7 @@ export CGO_CFLAGS+=" -m64 -mtune=generic -fcf-protection=full"
 export GOPROXY=direct
 
 LDFLAGS="-X %{ld_libpod}/define.buildInfo=${SOURCE_DATE_EPOCH:-$(date +%s)} \
+         -X "%{ld_libpod}/define.buildOrigin=%{build_origin}" \
```

That does not escape `"`, I assume this works:
```suggestion
         -X \"%{ld_libpod}/define.buildOrigin=%{build_origin}\" \
```

---

ugh, I did that change on the fedora rpm repo and didn't include here. My bad.